faq/faq.md

```diff
@@ -0,0 +1,65 @@
+<!--
+\-\--layout: page title: FAQ permalink: /faq/ order: 4 feature-img:
+\"examples/img/hero-bg.jpg\" feat_img_size: small \# Copyright (c) 2018,
+Intel Corporation. \# Licensed under Creative Commons Attribution 4.0
+International License \# <https://creativecommons.org/licenses/by/4.0/>
+\-\--
+-->
+
+# Hyperledger Sawtooth FAQ 
+
+---
+
+<!--
+========================
+-->
+
+Answers to frequently asked questions for Hyperledger Sawtooth. Many
+questions and answers originated from the \#sawtooth Rocket.Chat channel
+at <https://chat.hyperledger.org/channel/sawtooth>
+
+---
+
+## Sawtooth FAQ Sections
+
+-   [Hyperledger Sawtooth in General](sawtooth)
```

Comment:
These links are broken. You can test the site by running "just run" (after installing 'just') and pointing the browser to http://localhost:4000/.

The correct way to write these particular lines is:

```
 -   [Hyperledger Sawtooth in General]({% link faq/sawtooth.md %})
```